roadmap/implementers-guide/src/types/overseer-protocol.md

```diff
@@ -48,14 +50,17 @@ Messages to and from the availability store.
 
 ```rust
 enum AvailabilityStoreMessage {
-	/// Query the PoV of a candidate by hash.
-	QueryPoV(Hash, ResponseChannel<PoV>),
+	/// Query the AvailableData of a candidate by hash.
+	QueryPoV(Hash, ResponseChannel<Option<AvailableData>>),
```

Comment:
`type AvailableData` is not yet defined.

Reply:
defined in availability.md